app/controllers/recipe_revisions_controller.rb

```diff
@@ -4,27 +4,28 @@ class RecipeRevisionsController < ApplicationController
 
   def index
     @user             = @recipe.user
-    @revisions        = RecipeRevision.where(:recipe_id => @recipe.id)
+    @revisions        = RecipeRevision.where(recipe_id: @recipe.id)
     @recipe_permalink = @recipe.slug
   end
 
   def show
-    revision = RecipeRevision.find(params[:id])
+    @revision = RecipeRevision.find(params[:id])
 
-    if revision.revision > 1
-      previous_revision = RecipeRevision.where(:recipe_id => revision.recipe_id,
-                                               :revision => revision.revision - 1).first
-
-      @diff = Differ.diff_by_line(revision.body, previous_revision.body).format_as(:html).html_safe
+    if params[:target_id].present?
```

Comment:
@dpick you can compare different to a different revision from: your own recipe, from forks, from recipes you forked from. thats why I came up with target (ie. target revision to compare with). what do you think about:
`compare_recipe_revision`
`target_recipe_revision`

imagine the case where that you found a recipe for the same meal from a different user, wouldn't it be a good usecase to compare this one to yours aswell? I wouldn't put security checks in that place, but rather use some authorization in general (like cancan(can)).